src/handlers/indexer/handle-mint-event.ts

```diff
@@ -0,0 +1,73 @@
+import {TransactionReceipt, TransactionResponse} from '@ethersproject/abstract-provider'
+
+import {NetworkMonitor} from '../../utils/network-monitor'
+
+import SqsService from '../../services/sqs-service'
+import {networkToChainId} from '../../utils/utils'
+import {PayloadType, SqsMessageBody} from '../../types/sqs'
+
+async function handleMintEvent(
+  networkMonitor: NetworkMonitor,
+  transaction: TransactionResponse,
+  network: string,
+  tags: (string | number)[],
+): Promise<void> {
+  const receipt: TransactionReceipt | null = await networkMonitor.getTransactionReceipt({
```

Comment:
@BravoNatalie @alexanderattar 

So i think we can push more work to job consumer. The CLI only has to detect the new event and send a job. That way we simplify the CLI to be a block / event parser.